src/bidmanager.js

```diff
@@ -249,12 +249,37 @@ function processCallbacks(callbackQueue) {
   if (utils.isArray(callbackQueue)) {
     for (i = 0; i < callbackQueue.length; i++) {
       var func = callbackQueue[i];
-      func.call(pbjs, pbjs._bidsReceived);
+      func.call(pbjs, pbjs._bidsReceived.reduce(groupByPlacement, {}));
```

Comment:
@protonate 
Is there a way to reduce _bidsReceived by the `requestId` on `this` request? With this implementation we get an expanding list of bids back each time a requestBids is fired...